submodules/users/views/creation.html

```diff
@@ -1,34 +1,38 @@
 <div id="creation_user_dialog">
-	<form class="form-horizontal small-labels" id="form_user_creation">
-	<div>
-		<div class="control-group">
-			<label class="control-label" for="first_name"><i class="fa fa-user"></i></label>
-			<div class="controls">
-				<input required class="input-small" type="text" id="first_name" name="user.first_name" placeholder="{{i18n.users.dialogCreationUser.firstName}}"></input>
-				<input required class="same-line input-small" type="text" name="user.last_name" placeholder="{{i18n.users.dialogCreationUser.lastName}}"></input>
-			</div>
+	<form class="form-horizontal clearfix" id="form_user_creation">
+		<div class="row-fluid user-section-title">
+			<h3>{{i18n.users.dialogCreationUser.createUser}}</h3>
 		</div>
-
-		<div class="control-group">
-			<label class="control-label" for="email"><i class="fa fa-key"></i></label>
-			<div class="controls">
+		<div class="row-fluid">
+			<div class="span6 to-left">
+				<label class="control-label" for="first_name">{{i18n.users.dialogCreationUser.firstName}}</label>
+				<input required type="text" id="first_name" name="user.first_name" placeholder="{{i18n.users.dialogCreationUser.firstName}}" />
+			</div>
+			<div class="span6 to-right">
+				<label class="control-label" for="email">{{i18n.users.dialogCreationUser.loginEmail}}</label>
 				<input required type="email" name="user.username" id="username" placeholder="{{i18n.users.dialogCreationUser.loginEmail}}"/>
 			</div>
 		</div>
-
-		{{#unless _whitelabel.hide_user_passwords}}
-			<div class="control-group">
-				<label class="control-label" for="password"><i class="fa fa-unlock"></i></label>
-				<div class="controls">
-					<input required type="password" name="user.password" id="password" placeholder="{{i18n.users.dialogCreationUser.password}}"/>
+		<div class="row-fluid">
+			<div class="span6 to-left">
+				<label class="control-label" for="first_name">{{i18n.users.dialogCreationUser.lastName}}</label>
```

Comment:
Label `for` is related to the wrong input
```suggestion
				<label class="control-label" for="last_name">{{i18n.users.dialogCreationUser.lastName}}</label>
```

Also, you will need to set the `id` attribute for the input.